guides/jackson/integrations/bubble.md

```diff
@@ -35,7 +35,7 @@ Next step is to configure the plugin. Make sure you add the following values in
 
 - `Client Secret Verifier`: This is optional. If you have configured a client secret verifier in SAML Jackson, you can add it here. If you have not configured a client secret verifier, you can leave this field blank.
 
-![Configure the plugin](/img/bubble-io/step-1.png)
+![Configure the plugin](./bubble/step-1.png)
```

Comment:
Should we copy these to `images`?

Reply:
Moving ahead we should keep assets with markdown, docusaurus will handle it automatically.